ui/apps/platform/src/reducers/telemetryConfig.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
Is this needed? If so, could it be a single line lint disable instead?

Reply:
Nope not even needed, updated